tests/end_to_end/candid_rpc/class_syntax/cycles/test/test.ts

```diff
@@ -7,19 +7,21 @@ import { getTests } from 'cycles_end_to_end_test_functional_syntax/test/tests';
 import { createActor as createCyclesActor } from './dfx_generated/cycles';
 import { createActor as createIntermediaryActor } from './dfx_generated/intermediary';
 
-const cyclesCanister = createCyclesActor(getCanisterId('cycles'), {
+const cyclesCanisterName = 'cycles';
+const cyclesCanister = createCyclesActor(getCanisterId(cyclesCanisterName), {
     agentOptions: {
         host: 'http://127.0.0.1:8000'
     }
 });
 
+const intermediaryCanisterName = 'intermediary';
 const intermediaryCanister = createIntermediaryActor(
-    getCanisterId('intermediary'),
+    getCanisterId(intermediaryCanisterName),
     {
         agentOptions: {
             host: 'http://127.0.0.1:8000'
         }
     }
 );
 
-runTests(getTests(cyclesCanister, intermediaryCanister));
+runTests(getTests(cyclesCanister, intermediaryCanister), cyclesCanisterName);
```

Comment:
Why aren't they both passed in?

Reply:
I don't know... I guess I mistakenly thought that there would be errors coming from all of the tests that needed to be updated like that...

Reply:
fixed